app/src/main/java/co/copperhead/pdfviewer/PdfViewer.java

```diff
@@ -40,13 +61,14 @@
 
     private WebView mWebView;
     private Uri mUri;
-    int mPage;
-    int mNumPages;
+    public int mPage;
```

Comment:
I've moved all fragments to a separate package.
In the future once this project expands it's better to keep things sorted, for better overview.
In case more activities will be added, i'd also suggest creating a separate package for them.